pom.xml

```diff
@@ -364,6 +350,29 @@
         <auto-value-annotation.version>1.6.6</auto-value-annotation.version>
         <auto-value.version>1.4</auto-value.version>
       </properties>
+      <build>
+        <plugins>
+          <plugin>
+            <artifactId>maven-compiler-plugin</artifactId>
+            <configuration>
+              <annotationProcessorPaths>
+                <path>
+                  <groupId>com.google.auto.value</groupId>
+                  <artifactId>auto-value</artifactId>
+                  <version>${auto-value.version}</version>
+                </path>
+                <!--
+                There is currently no available version of auto-service-annotations
```

Comment:
The Java 7 CI failed. Perhaps we should ask @netdpb to push one compiled for Java 7?

Reply:
The java 7 build failed because of a timeout from maven central.

I don't know if we specifically need a release for Java 7 as we perform release builds with Java 8 and a 1.7 target.